snippets/deepFreeze.md

```diff
@@ -0,0 +1,38 @@
+### deepFreeze
+
+Deep freezes an object.
+
+Calls `Object.freeze(obj)` recursively on all unfrozen properties of obj that are `typeof` function or object.
```

Comment:
Phrasing improvement suggestion:

Call `Object.freeze(obj)` recursively on all unfrozen properties of `obj` that are identified as functions or objects, using `typeof`.

Just my opinion, there might be an even better way to phrase this.